charts/astronomer/templates/prisma/prisma-configmap.yaml

```diff
@@ -18,7 +18,21 @@ data:
     databases:
       default:
         connector: postgres
+        {{ if and (.Values.prisma.backendSecretName) (not .Values.prisma.backendConnection) }}
+        host: $PRISMA_DB_HOST
+        port: $PRISMA_DB_PORT
+        user: $PRISMA_DB_USER
+        password: $PRISMA_DB_PASSWORD
+        database: {{ printf "%s_houston" (.Release.Name | replace "-" "_")  }}
+        {{ else if and (.Values.prisma.backendConnection) (not .Values.prisma.backendSecretName) }}
+        host: {{ .Values.prisma.backendConnection.host }}
+        port: {{ .Values.prisma.backendConnection.port }}
+        user: {{ .Values.prisma.backendConnection.user }}
+        password: {{ .Values.prisma.backendConnection.pass }}
+        database: {{ printf "%s_houston" (.Release.Name | replace "-" "_")  }}
```

Comment:
@ianstanton since we duplicate this in both blocks, we could assign a variable up top first, and use it down here, like we do over here for example: https://github.com/astronomer/airflow-chart/blob/master/templates/scheduler/scheduler-deployment.yaml#L5-L15.